pkg/pxc/app/statefulset/proxysql.go

```diff
@@ -82,12 +82,6 @@ func (c *Proxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaX
 				Name:  "PXC_SERVICE",
 				Value: c.labels["app.kubernetes.io/instance"] + "-pxc",
 			},
-			{
```

Comment:
remove and add vars only during the update to 1.5.0

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/452/commits/83ec682d736435e54368f4030320936ce2644520

Reply:
please don't delete MYSQL_ROOT_PASSWORD var if version <= 1.4.0 😞